util/src/journaldb/archivedb.rs

```diff
@@ -26,6 +26,8 @@ use kvdb::{Database, DBTransaction, DatabaseConfig};
 #[cfg(test)]
 use std::env;
 
+const AUX_FLAG: u8 = 255;
```

Comment:
I've added a description of this const and your comment. Also added this issue https://github.com/ethcore/parity/issues/1455